src/main/java/fr/esgi/al5_2/Tayarim/mappers/AdministrateurMapper.java

```diff
@@ -0,0 +1,48 @@
+package fr.esgi.al5_2.Tayarim.mappers;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.AdministrateurCreationDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.AdministrateurDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireCreationDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireDTO;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import lombok.NonNull;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AdministrateurMapper {
+
+  public static Administrateur creationDtoToEntity(
+      @NonNull AdministrateurCreationDTO administrateurCreationDTO) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'administrateurCreationDTO' must contain no more than '1' consecutive capital letters.